### DISCUSSION TITLE
Media Composer jog issues

### DISCUSSION THREAD

**Discussion by mattbedits** [CommandPost/CommandPost#3302] [General]
Can anyone who has successfully mapped the jog wheel in Avid share their settings? I am having massive issues trying to map the correct keys and settings inside Avid and variables with commandpost to make it useable.

Thanks so much!

**Comment by latenitefilms** [comment]
I believe Marc is using the Speed Editor with CommandPost in Avid Media Composer:

https://twitter.com/marcplanb

**Comment by rcsbardin** [comment]
It's not the same behavior as you might want but I was pretty excited by the setup I created today.

Program Shuttle button to trigger Bank 1 (name bank 1 shuttle mode)
Bank 1 settings - Shuttle LED always on. Jog wheel Relative (fine) / Very sensitive / Turn LeftMouse Scroll Wheel 1 line down plus control plus option / Turn Right Mouse Scroll Wheel 1 line up plus control plus option
Program Jog to Bank 2 (named bank 2 Jog mode)
Bank 2 settings - Jog LED always on. Jog wheel Relative (fine) / sensitivity 5 / Turn LeftMouse Scroll Wheel 10 line down plus control / Turn Right Mouse Scroll Wheel 10 line up plus control (not sure of the difference between 10 and 1 line tbh)
Program Scroll to Bank 3 (name bank 3 Scroll mode)
Bank 3 settings - Scroll LED always on. Jog wheel relative fine. Default sensitivity (10) - turn left - whatever your shortcut in avid is for Go to Previous event (or trim might work), turn right - shortcut for Next event (or trim?)
And then make sure you copy your other commands like insert etc across all banks.

Hope that is helpful. I don't know how to really get variable speeds without diving into coding...and I need to actually be editing the film. Maybe this will help someone make an even better version though!

**Reply by strykedits** [reply]
I'm posting a video today for Premiere that might help you.   -Stryker

**Reply by strykedits** [reply]
Here is the video:  
https://www.youtube.com/watch?v=PcyfasCqSJU

**Reply by rcsbardin** [reply]
Oh yes, can't wait to try this. I hopped back into Resolve for a different project and realized the native behaviors are very different from what I'd created for Avid media composer. Hope to test out this jkl version you've done. I hadn't ventured into snippets yet either so that was helpful to see how it is used. Thanks for sharing!

**Reply by strykedits** [reply]
Tell us if this works in MC.   Ironically I used Media Composer for 10 years but not in the last 2.   Thanks.
